### ISSUE TITLE
Question About Result

### ISSUE THREAD

**Issue by zhangguanghui1** [JiaRenChang/PSMNet#19]
@JiaRenChang  Hi, The model "pretrained_model_KITTI2015.tar" I download from your link. It is trained on SenceFlowData with a constant learning rate of 0.001 for 10 epochs firstly , and then finetune it on KITTI 2015 training dataset for 300 epochs . And the learning rate of this fine-tuning began at 0.001 for the first 200 epochs and 0.0001 for the remaining 100 epochs.  right?  Because I trained the model  as your paper said with your code, I cannot get the same result as "pretrained_model_KITTI2015.tar". It is worse than yours, Thank you very much!!!

**Comment by JiaRenChang** [comment]
@zhangguanghui1 
The model we uploaded was trained for 1000 epochs, and it was also described in the paper.

**Comment by zhangguanghui1** [comment]
@JiaRenChang Okay, Thank you so much!

**Comment by havefun007** [comment]
Hey, I get some trouble when i try to decompress the pretrained model file, there is something wrong with the tar file, can you help me?@JiaRenChang  @zhangguanghui1

**Comment by JiaRenChang** [comment]
@havefun007 
You don't need to decompress the .tar file, and just use torch.load() to load it.

**Comment by havefun007** [comment]
Oh....，Thanks @JiaRenChang
